src/Artemis.UI/Screens/Debugger/Tabs/Render/RenderDebugView.axaml

```diff
@@ -30,7 +30,9 @@
                         ClipToBounds="True"
                         Focusable="True"
                         VerticalAlignment="Stretch"
-                        HorizontalAlignment="Stretch">
+                        HorizontalAlignment="Stretch"
```

Comment:
I agree the pan/zoom functionality is perhaps a bit pointless, though I added it for really big surfaces.
But if we're going to remove this then wrapping in in a disabled ZoomBorder is pretty pointless :P